css/style.css

```diff
@@ -845,17 +854,11 @@ button:hover {
 
 @media (max-width: 480px) {
 	/* hide board actions on mobile */
-	#board-actions {
+	#board-header-controls {
 		display: none !important;
```

Comment:
@pixelipo @juliushaertl what do you think about this suggestion
![screenshot_20170522_181847](https://cloud.githubusercontent.com/assets/16517095/26318423/53f8f17e-3f1b-11e7-9f22-bfb6abe853b9.png)